### DISCUSSION TITLE
如何实现在有个配置文件中, 让一个入端口对应一个出端口

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#764] [Q&A]
我目前的使用模式是这样的: 我在浏览器上使用SwitchyOmega插件来管理代理端口, 监听tcp/10805 使用http协议, 监听tcp/10804使用socks协议, 使用唯一一个出接口: tcp/10801, 出接口就对应我的VPS, 然后在浏览器上实现不同域名能进行相应的代理. 现在我的需求是这样的, 我想再创建一个出接口, 比如tcp/10802, 然后让入接口10804的流量都走tcp/10801, 让tcp/10805的流量都走10802, 让一个入接口对应一个出接口. 这样该如何实现?

**Comment by Loyalsoldier** [comment]
inboudTag 和 outboundTag 一一对应即可。

**Reply by ghost** [reply]
能多问下, 具体上, 配置文件上格式该如何写?

**Reply by Loyalsoldier** [reply]
客户端参考[这里](https://github.com/Loyalsoldier/v2ray-rules-dat#%E8%87%AA%E7%94%A8-v2ray-%E5%AE%A2%E6%88%B7%E7%AB%AF%E9%85%8D%E7%BD%AE%E4%BB%85%E4%BE%9B%E5%8F%82%E8%80%83%E8%AF%B7%E6%A0%B9%E6%8D%AE%E8%87%AA%E8%BA%AB%E9%9C%80%E6%B1%82%E9%85%8C%E6%83%85%E4%BF%AE%E6%94%B9)，服务端大同小异。

**Reply by ghost** [reply]
好的， 谢谢， 我现在就先试一下。

**Reply by ghost** [reply]
好了， 可以使用了， 谢谢。
